docs/sources/flow/reference/components/beyla.ebpf.md

```diff
@@ -0,0 +1,250 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/beyla.ebpf/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/beyla.ebpf/
+description: Learn about beyla.ebpf
+title: beyla.ebpf
+---
+
+# beyla.ebpf
+
+The `beyla.ebpf` component is used a wrapper of [Grafana Beyla][] which collects metrics and traces from eBPF programs running on the host. The component can be configured to collect data from a specific port or executable name, and other criteria from Kubernetes metadata. The component can  be scraped by Prometheus and can send traces to an OpenTelemetry Collector.
+
+{{< admonition type="note" >}}
+To run this component, the Agent requires administrative (`sudo`) privileges, or at least it needs to be granted the `CAP_SYS_ADMIN` capability.
+{{< /admonition >}}
+
+[Grafana Beyla]: https://github.com/grafana/beyla
+
+## Usage
+
+```river
+beyla.ebpf "LABEL" {
+ 
+}
+```
+
+## Arguments
+
+`beyla.ebpf` supports the following arguments:
+
+Name | Type | Description                                               | Default | Required
+---- | ---- |-----------------------------------------------------------| ------- | --------
+`open_port` | `string` | The port of the running service for Beyla automatically instrumented with eBPF. | | no
+`excutable_name` | `string` | The name of the executable to match. | | no
+
+
+`open_port` accepts a comma-separated list of ports (for example, 80), and port ranges (for example, 8000-8999). If the executable matching only one of the ports in the list, it is considered to match the selection criteria.
```

Comment:
nit: your example of a coma-separated list of ports is not a comma-separated list of ports :)